### PR TITLE
TabHandleView: Add sub view to close-tab icon.

### PR DESCRIPTION
**Pull request by GokhanTurunc**
/cc @gokmen @sinan @fatihacet @szkl

### CODE REVIEW

**Comment by fatihacet** [comment]
@sinan this is kd core level change and would be better if it's review and merged by you.

File: lib/components/tabs/tabview.coffee [hunk 1]
```diff
@@ -257,10 +257,12 @@ module.exports = class KDTabView extends KDScrollView
   # DEFAULT ACTIONS
   handleClicked: (event, handle) ->
 
-    {pane} = handle.getOptions()
+    { pane }  = handle.getOptions()
+    target    = $ event.target
 
     # fixme: make close icon a kdview and check the view instead.
```
Comment (sinan) [inline lib/components/tabs/tabview.coffee]: instead of adding another nested element, let's do it as I commented here over a year ago :)

**Comment by GokhanTurunc** [comment]
@sinan could you check again this pr? I've pushed something according to your feedbacks. But I didn't check the `view` because the `target` gives me what I want.

**Comment by fatihacet** [comment]
@GokhanTurunc this can be a breaking change and we have to be sure there won't be a side effect of this PR. Could you test it more? We can discuss about it more. Ping me on slack.

**Comment by GokhanTurunc** [comment]
ping

**Comment by GokhanTurunc** [comment]
ping

File: lib/components/tabs/tabview.coffee [hunk 2]
```diff
@@ -257,15 +258,13 @@ module.exports = class KDTabView extends KDScrollView
...
-    {pane} = handle.getOptions()
+    { pane } = handle.getOptions()
+    @showPane pane
 
-    # fixme: make close icon a kdview and check the view instead.
-    if $(event.target).hasClass 'close-tab'
-      @blockTabHandleResize = yes
-      @removePane pane
-      return no
+  handleCloseIcon: (pane) ->
```
Comment (fatihacet) [inline lib/components/tabs/tabview.coffee]: can you change method name to `handleCloseAction`

**Comment by GokhanTurunc** [comment]
ping

**Comment by fatihacet** [comment]
LGTM :+1:

waiting for @sinan to merge it.

**Comment by sinan** [comment]
bump up version, and we'll be good.

**Comment by GokhanTurunc** [comment]
ready.

**Comment by sinan** [comment]
@GokhanTurunc please apply the same thing to the master branch as well.

**Comment by GokhanTurunc** [comment]
I've created a pull request for this: 
The pull request: https://github.com/koding/koding/pull/5222

**Comment by GokhanTurunc** [comment]
@sinan i'll do it.

**Pull request merged**